app/assets/stylesheets/components/_navbar.scss

```diff
@@ -11,29 +11,138 @@
 .navbar-rv {
   display: flex;
   justify-content: space-between;
-  background-color: darkgrey;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  transition: all .50s ease;
+
+  img {
+    margin-left: 8px;
+  }
 }
+
+
 .navbar-logo {
   padding: 0;
 }
 
 .navbar-categories {
   display: flex;
-  align-items: center;
+  align-items: flex-end;
+  padding-bottom: 16px;
 }
 
 
 .navbar-about {
   display: flex;
-  align-items: center;
-  margin: 8px;
+  align-items: flex-end;
+  padding-bottom: 16px;
+  margin-right: 8px;
 }
 
-.navbar-item {
-  padding: 5px 8px;
+.navbar-item , .mobile-item{
+  padding: 3px 8px;
 }
 
-.navbar-categorie:hover {
-  border-bottom: solid 1px black;
+.navbar-item:hover {
+  padding-bottom: 0px;
+  border-bottom: solid 3px #e9815b;
 }
 
+.navbar-hover {
+  background-color: white;
+  box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.2);
+}
+
+// -- Mobile version
+
+@media (max-width: 768px) {
+  .navbar-about, .navbar-categories, .navbar-rv{
+    display: none;
+  }
+
+  .background-mobile {
+    z-index: 4;
+    position: fixed;
+    top: 0;
+    display: block;
+    background: white;
+    border: solid 1px gray;
+    height: 100vh;
+    width: 200px;
+    margin-left: -300px;
+    transition: all 1s ease;
+
+    &.menu-active{
```

Comment:
I understand the need to an empty line before the `&.class` (don't forget space before { ) but not necessarily before and after each class in the same big group (here, media query). I suggest deleting all empty lines except the one on top of `&.class`.